src/api/buildSwapTransaction.ts

```diff
@@ -45,6 +45,17 @@ export async function buildSwapTransaction(
     };
   }
 
+  // Adjust slippage for V1 API (aggregator)
+  // V1 expects slippage in tenths of a percent (e.g., 30 = 3%)
+  if (params.useAggregator && params.maxSlippage) {
+    apiParams = {
+      slippagePercentage: (
+        Number.parseFloat(params.maxSlippage) * 10
+      ).toString(),
+      ...apiParams,
```

Comment:
Why are we doing this? this seems odd.

You just can do
`apiParams.slippagePercentage = potato`
no need all the extra code

Reply:
Updated to:
```

    let slippage = Number(params.maxSlippage);
  
    slippage = slippage * 10;
    
    apiParams.slippagePercentage = slippage.toString();
```